README.md

```diff
@@ -17,10 +17,9 @@ The collective effort of the Project Pokémon (and other) communities to preserv
 * For Gen 4, 5, 6 and 7 (3DS) events, use the [PKHeX Mystery Gift feature](https://projectpokemon.org/home/tutorials/save-editing/using-pkhex/importing-wondercards-r29/)
 * For Gen 7 (Switch), 8 and 9 events, use the [Switch Gift Data Manager](https://projectpokemon.org/home/forums/topic/62491-switch-gift-data-manager-import-wondercards-into-switch-games-by-faking-bcat-packages/)
 
-#### • Import Raid blocks with PKHeX
-1. Follow our [tutorial](https://projectpokemon.org/tutorials/save-editing/gen-8_157/block-editing-r106/) to import the files into the corresponding blocks into your save file.
-1. Sword and Shield Raid blocks: `KDropRewards`, `KDaiEncount`, `KBonusRewards`, `KNormalEncount`, `KNormalEncountRigel1`, `KNormalEncountRigel2`. 
-1. Scarlet and Violet Raid blocks: `KBCATEventRaidIdentifier`, `KBCATFixedRewardItemArray`, `KBCATLotteryRewardItemArray`, `KBCATRaidEnemyArray`, `KBCATRaidPriorityArray`. 
+#### • Import Raid Events
+* For Sword/Shield raids, follow our [PKHeX Block Editing tutorial](https://projectpokemon.org/tutorials/save-editing/gen-8_157/block-editing-r106/). The blocks to be imported are labeled as follows: `*Object KDropRewards`, `*Object KDaiEncount`, `*Object KBonusRewards`, `*Object KNormalEncount`, `*Object KNormalEncountRigel1`, `*Object KNormalEncountRigel2`. 
+* For Scarlet/Violet raids, follow the [tutorial for importing Poké Portal News](https://projectpokemon.org/home/tutorials/save-editing/gen-9/gen-9-specific-edits-importing-poké-portal-news-raid-events-r124/).
```

Comment:
It would make sense to keep the block names like for Scarlet and Violet like you did for Sword and Shield or remove them for Sword and Shield too.

Reply:
Keeping the block names for Scarlet and Violet would be redundant since we already have them listed in the tutorial page on the site. All the infos needed for the importing stage in Scarlet & Violet is available in the tutorial page.

On the countrary, for Sword and Shield we don't have a tutorial page specifically for the raid imports. The tutorial linked is for Block Editing, and does not cover specifically the blocks needed for the raids. Hence why we need to list the block names here in the readme.

Reply:
I remembered that the tutorial was using the blocks from raids but didn't notice it didn't specify the locations for the ones other than the normal encount one.